plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -296,13 +297,29 @@ public final void testGoogleApisExportVersions() throws IOException {
 
   private void checkDependencyDirectives(
       String attributeName, String prefixToCheck, String versionString) throws IOException {
+    checkDependencyDirectives(attributeName, prefixToCheck, versionString, Lists.newArrayList());
+  }
+  
+  private void checkDependencyDirectives(
```

Comment:
This test is inherited by other packages. It basically confirms that dependency with prefix X has version[ range] Y. These are hardcoded in this same class. 
If we follow the [suggestion](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3716/files#r1135615733) in the non-cached credentials PR, then this change may have to be reverted.